core/templates/dev/head/app.js

```diff
@@ -44,7 +44,8 @@ oppia.constant('OBJECT_EDITOR_URL_PREFIX', '/object_editor_template/');
 // NOTE TO DEVELOPERS: This should be synchronized with the value in feconf.
 oppia.constant('ENABLE_STRING_CLASSIFIER', false);
 // Feature still in development.
-oppia.constant('ENABLE_HINT_EDITOR', false);
+oppia.constant('ENABLE_HINT_EDITOR', true);
+oppia.constant('ENABLE_FALLBACK_EDITOR', false);
```

Comment:
I'm not sure why we have this instead of just deleting the fallbacks stuff. But I guess it may make it easier to revert if needed, so we can leave it in for one release.

That said, you'll need to hide the control in the Settings tab as well.